### ISSUE TITLE
Improved UI

### ISSUE THREAD

**Issue by DeveloperAspire** [DeveloperAspire/developers-fact-card#27]
The UI is currently laggy and shitty, it needs improvement.

* Font
* Color
* Feel and look
* Responsiveness
* General design
* More features

**Comment by Nagendra88688** [comment]
@DeveloperAspire can i pick up on this task?

**Comment by DeveloperAspire** [comment]
@Nagendra88688 Hi! Sure you can take on it.
Do you have a design in mind?

**Comment by Nagendra88688** [comment]
@DeveloperAspire Yes, I'm thinking of using better fonts can make cards look more interactive. And the profile photo of each card should also be interactive on hovering on it.

**Comment by DeveloperAspire** [comment]
@Nagendra88688 Sounds like a plan, Chief! Carry on! Make your PR and I will drop feedback. Thanks for picking this up!

**Comment by Nagendra88688** [comment]
okay, thank you :)

**Comment by Nagendra88688** [comment]
hey @DeveloperAspire 
I've created a pull request with some changes, please review and let me know.
link of pull request ->   https://github.com/DeveloperAspire/developers-fact-card/pull/29

**Comment by Nagendra88688** [comment]
Any follow up?
@DeveloperAspire

**Comment by DeveloperAspire** [comment]
@Nagendra88688 Thank you for this Chief! I will definitely go through it and let you know. Sorry for the delay. Work has been crazy.

**Comment by Nagendra88688** [comment]
No problem @DeveloperAspire , please let me know your feedback :)
